client/src/LayoutComponents/Cohorts/Cohorts.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import "./Cohorts.css";
+import CohortCard from "../../SharedComponents/CohortCard";
+import Slideshow from "../../SharedComponents/Slideshow";
+import axios from "axios";
+
+class Cohorts extends Component {
+  componentDidMount() {
+    const cityName = this.props.match.params.city;
+    axios
+      .get(`/cohorts/${cityName}`)
+      .then((response) => {
+        console.log(response.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+  render() {
+    const cityName = this.props.match.params.city;
+    // console.log(
+    //   this.props.history.location.state.data,
+    //   "this is cohorts page test "
+    // );
+
+    return (
+      <div>
```

Comment:
@bayan-404 Here is where you could add a `cohorts` className: `<div className='cohorts'>`